android/app/src/main/java/org/fossasia/openevent/core/auth/profile/EditProfileActivity.java

```diff
@@ -82,7 +98,21 @@ protected void onCreate(Bundle savedInstanceState) {
             showUserData(user);
         });
 
-        avatar.setOnClickListener(v -> showFileChooser());
+        Observable<Boolean> picObserver = Observable.just(checkReadExternalPermission());
+        picObserver.subscribeOn(Schedulers.io())
+                .observeOn(AndroidSchedulers.mainThread())
+                .subscribe(picBool -> {
+                    if (picBool) {
```

Comment:
checking picBool to be true to set permission granted true makes no sense
also please use standard naming conventions